public/api/docs/v2/swagger.yaml

```diff
@@ -353,7 +383,223 @@ components:
               nullable: true
               type: boolean
               example: true
-    IDAttribute:
-      description: The unique ID of the resource.
-      type: uuid
-      example: d0b4a32e-a272-489e-b30a-cb17131457fc
+    ApplicationsCsvResponse:
```

Comment:
not sure why this is rendering correctly, see https://npq-registration-review-1333-web.test.teacherservices.cloud/api/docs/v2